source/frontend/src/features/mapSideBar/acquisition/tabs/stakeholders/update/UpdateStakeHolderForm.tsx

```diff
@@ -87,6 +87,7 @@ export const UpdateStakeHolderForm: React.FunctionComponent<IUpdateStakeHolderFo
                                 View={ContactInputView}
                               ></ContactInputContainer>
                             </Col>
+                            {interestHolder.contact?.organization?.organizationPersons?.length}
```

Comment:
this looks like that number I mentioned the other day.

Reply:
weird, no idea how that got in there.